apps/server/src/infra/s3-client/s3-client.adapter.spec.ts

```diff
@@ -351,9 +448,15 @@ describe('S3ClientAdapter', () => {
 				const setup = () => {
 					const { pathToFile, bucket } = createParameter();
 					const filePath = 'directory/test.txt';
-					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-					// @ts-ignore
-					client.send.mockResolvedValueOnce({ Contents: [{ Key: filePath }] });
+
+					const expectedResponse: ListObjectsV2CommandOutput = {
```

Comment:
In eine Test Factory function verschieben. Gerne auch im gleichen File. Aber einmalig.